src/dbg/stackinfo.cpp

```diff
@@ -195,7 +239,7 @@ void stackgetcallstack(duint csp, std::vector<CALLSTACKENTRY> & callstackVector,
             return;
         }
         callstackVector.clear();
-        return;
+        //return;

```

Comment:
It is because updating the setting in the GUI (and then clearing the cache via `stackupdatesettings()`) will empty the call stack view. Any way not to update but not empty that view?